frontend/src/pages/Landing/ReportMenu.js

```diff
@@ -43,12 +43,12 @@ function ReportMenu({
   };
 
   return (
-    <span>
+    <span className="position-relative">
```

Comment:
Affects positioning of `position-absolute` child elements. This helps us position the menu without manually setting `left: 85px`.